src/styles/blocks/rating.scss

```diff
@@ -0,0 +1,13 @@
+.rating {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-inline: $standart-inline-pad;
+  padding-bottom: 24px;
```

Comment:
Consider defining the padding-bottom value as a variable in a separate file to adhere to the task requirements for maintainability.